plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -370,7 +370,7 @@ class SetupAuthenticationDialog(
                         reason = "InvalidCredentials",
                         isReAuth = false
                     )
-                    return
```

Comment:
yeah i think we're trying to avoid an unhandled error, so not sure if this is how we go about this